lib/chrome-out.js

```diff
@@ -5,13 +5,15 @@ const unmirror = require('chrome-unmirror');
 /*
  * Pipes chrome console messages to stdout
  */
-module.exports = function chromeOut (log, Runtime) {
+module.exports = function chromeOut (log, Runtime, options) {
 
   Runtime.exceptionThrown((exception) => {
-    log.error('[chrome-exception]', exception);
+    if (!options.ignoreExceptions) log.error('[chrome-exception]', exception);
```

Comment:
i'll have to check the linter config, but this shouldn't have passed. please use the https://eslint.org/docs/rules/brace-style#stroustrup style here:

```
if () {
 ...
}
```